packages/slice-machine/components/FormFields/Input.tsx

```diff
@@ -36,25 +43,22 @@ const getFieldStyles = (
           boxShadow:
             "0 0 0 3px rgba(255, 165, 0, 0.2), inset 0 1px 2px rgba(255, 165, 0, 0.2)",
         },
-      }
-    : defaultFieldStyles;
-
-  return fieldStyles;
+      };
+    }
+    case InputFieldStyles.DEFAULT: {
```

Comment:
no need for that case, it will fall in the default if this one is not there.

Reply:
ah good point, removed